scripts/checkpoint_converters/convert_mixtral_hf_to_nemo.py

```diff
@@ -108,6 +113,9 @@ def load_config(mixtral_config, tokenizer_path):
     # RMSNorm's epsilon.
     nemo_config.layernorm_epsilon = mixtral_config['rms_norm_eps']
     nemo_config.normalization = 'rmsnorm'
+    nemo_config.micro_batch_size = 1
```

Comment:
why this is necessary?

Reply:
was getting a micro_batch_size of 4 in the final config for some reason.